src/test/java/com/plaid/client/integration/banktransfer/AbstractBankTransferTest.java

```diff
@@ -64,6 +65,7 @@ public void setUpBankTransfer() throws Exception {
   protected abstract void bankTransferTest() throws AssertionError, Exception;
 
   @Test
+  @Ignore("Intermittently failing.")
```

Comment:
Should we get the BT team looking at these? Theoretically these shouldn't be failing, right?

Reply:
yea you're right, we should fix it. @bcambs09 any insight here?